pkgs/top-level/python-packages.nix

```diff
@@ -81,7 +81,9 @@ let
         pythonPath = [ ]; # Deprecated, for compatibility.
         requiredPythonModules = requiredPythonModules drv.propagatedBuildInputs;
       };
-    });
+    }) // {
+      overridePythonAttrs = drv.overrideAttrs;
+    };
```

Comment:
Problems:
- This is not composable: You can't `overridePythonAttrs` twice
- `overrideAttrs` doesn't support non-functions, so `.overridePythonAttrs {}` fails, leading to inconsistent behavior with the `overridePythonAttrs` that `makeOverridablePythonPackage` defines.

Reply:
I see. Any suggestion how this can be resolved by adding the override option after the original function (`stdenv.mkDerivation`) has already been called? This here also seems a case where https://github.com/NixOS/rfcs/pull/3 would be helpful.

Reply:
Usually this is done by calling the original function again, but I think this get really messy with multiple ways to override things. Maybe look at the implementation of `makeOverridable`